fe/fe-core/src/main/java/com/starrocks/sql/ast/ManualRefreshSchemeDesc.java

```diff
@@ -0,0 +1,14 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.sql.ast;
+
+import com.starrocks.catalog.RefreshType;
+
+public class ManualRefreshSchemeDesc extends RefreshSchemeDesc {
+
+    public ManualRefreshSchemeDesc() {
+        this.type = RefreshType.MANUAL;
```

Comment:
better to add a constructor in base class RefreshSchemeDesc like:
RefreshSchemeDesc(RefreshType typ) : this.type(type) { }
And just call super to init this field in all derived classes.